includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -72,6 +186,47 @@ protected function setup_info() {
 		);
 	}
 
+	/**
+	 * Gets the filters for retrieving publications for the current site.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Permutations for site hosts or URL.
```

Comment:
The function now returns a string.

```suggestion
	 * @return string Permutations for site hosts or URL.
```